broker-cli/commands/wallet.js

```diff
@@ -352,20 +352,44 @@ async function release (args, opts, logger) {
     if (!ACCEPTED_ANSWERS.includes(answer.toLowerCase())) return
 
     try {
-      await client.walletService.releaseChannels({ market, force })
-    } catch (e) {
-      logger.error(`Failed to release payment channels for ${market}`.red)
+      // We want to keep track if any returned channel had an error while attempting
+      // to be released so that we can display the proper error warnings below
+      let channelHasError = false
+
+      const { channels = [] } = await client.walletService.releaseChannels({ market, force })
+
+      if (!channels) {
+        throw new Error('No information was retrieved from the daemon')
+      }
+
+      logger.info(`Released Market: ${market}`)
```

Comment:
this log is a bit weird in the context of the logs that you do below, and that the user specifically requested this market be released.

Reply:
@treygriffith you think I should just remove it?